context-free.md

```diff
@@ -1,22 +1,23 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language is for anyone interested in art and context free grammars.
 
 ## What is easy to do in this language? Why is it easy?
-
+It is easy to generate recuring shapes.
```

Comment:
Not sure if you mean recurring or recursing here. I'd agree with both, but especially recursive shapes are made easy with ContextFree.